_scalingo_api/domains.md

```diff
@@ -14,17 +14,31 @@ result in the presence of one resource domain.
 **Keys attributes**
 
 {:.table}
-| field                | type                                                              |
-| -------------------- | ----------------------------------------------------------------- |
-| id                   | unique ID of the domain                                           |
-| name                 | hostname your want to associate with the app                      |
-| tlscert (read)       | subject of the submitted certificate                              |
-| tlscert (write)      | content of the SSL certificate you want to use                    |
-| tlskey (read)        | private key type and length                                       |
-| tlskey (write)       | content of the private key used to generate the certificate       |
-| ssl (read-only)      | flag if SSL with a custom certificate is enabled                  |
-| validity (read-only) | once a certificate has been submitted, display the validity of it |
-| canonical            | the domain is the canonical domain of this application            |
+| field                | type     | description                                                       |
+| -------------------- | -------- | ----------------------------------------------------------------- |
+| id                   | string   | unique ID of the domain                                           |
+| name                 | string   | hostname your want to associate with the app                      |
+| tlscert (read)       | string   | subject of the submitted certificate                              |
+| tlscert (write)      | string   | content of the SSL certificate you want to use                    |
+| tlskey (read)        | string   | private key type and length                                       |
+| tlskey (write)       | string   | content of the private key used to generate the certificate       |
+| ssl (read-only)      | bool     | flag if SSL with a custom certificate is enabled                  |
+| validity (read-only) | datetime | once a certificate has been submitted, display the validity of it |
+| canonical            | bool     | the domain is the canonical domain of this application            |
+| letsencrypt          | bool     | the domain is using a let's encrypt status                        |
```

Comment:
I don't really understand what this boolean is? What does it mean for a domain to use a Let's Encrypt status?